kernel/kernel-api/src/main/java/io/delta/kernel/internal/replay/ConflictChecker.java

```diff
@@ -141,6 +158,15 @@ public TransactionRebaseState(long latestVersion) {
         public long getLatestVersion() {
             return latestVersion;
         }
+
+        /**
+         * Return the latest CommitTimestamp of the table.
```

Comment:
s/CommitTimestamp/commit timestamp. Also, please clarify what this means for both ICT and non-ICT tables.